docs/source/release/v6.1.0/sans.rst

```diff
@@ -18,5 +19,18 @@ Improvements
 ------------
 
 - With SANSILLAutoProcess, the detector distance, the collimation position and the wavelength are appended to the names of the output workspaces (values are taken from the sample logs).
+- :ref:`SANSILLAutoProcess <algm-SANSILLAutoProcess>` has new property:
+  `StitchReferenceIndex` to denote the index of ws that should be a reference
+  for scaling during stitching
+
+Algorithms and instruments
+--------------------------
+
+Improvements
+############
+
+- :ref:`SANSILLAutoProcess <algm-SANSILLAutoProcess>` has new property:
```

Comment:
This note is duplicated above. Also add a note for Stitch1DMany itself in framework section.

Reply:
There is a note for Stitch1DMany in the framework section already, also poorly merged, with a missing line, which naturally will be corrected.